File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/pehrs.py
#!/usr/bin/python3

[l, r] = map(int, input().split())

if l == 0 and r == 0:
    print("Not a moose")
elif l == r:
    print("Even {}".format(l+r))
else:
    print("Odd {}".format(max(l,r)*2))
    

File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/pehrs.java
import java.util.*;

public class pehrs{
    public static void main(String[] args){
        Scanner io = new Scanner(System.in);
        int l = io.nextInt();
        int r = io.nextInt();
        if (l == 0 && r == 0) {
            System.out.println("Not a moose");

        } else {
            if (l == r ) {
                System.out.println("Even "+(l+r));
            } else {
                System.out.println("Odd "+Math.max(l, r)*2);
            }
        }
    }
}
